CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -1,5 +1,17 @@
 # Changelog 1.11
 
+## 1.11.1 - 2023-04-19
+
+### Changed
+
+- Document the limit of maximum receiver group number on a Linux Node. ([#4850](https://github.com/antrea-io/antrea/pull/4850), [@ceclinux])
```

Comment:
this sentence by itself doesn't make it clear that this is for the multicast feature

Reply:
Append with `for multicast`.